GRFC-TEMPLATE.md

```diff
@@ -2,21 +2,19 @@ Title
 ----
 * Author(s): [Author Name, Co-Author Name ...]
 * Approver: a11r
-* Status: {Draft, In Review, Ready for Implementation, Implemented}
 * Implemented in: <language, ...>
 * Last updated: [Date]
-* Discussion at: <google group thread> (filled after thread exists)
```

Comment:
I do think we need at least a notification to the grpc-io list. Yes, discussions rarely happen there, but the notification still seems fair. (This would be a bit different if GitHub had a way to watch for new PRs without also subscribing to those PRs.)

Also, removing the notification involved here seems over-stepping the sort of change we should do in this PR if we aren't going to go through a fuller process. Like, it seems such a change like removing the grpc-io threads needs a grpc-io thread. And we are currently creating threads, so it seems weird to remove it if this PR is to reflect the actual process.

Reply:
Should we include a back-reference to the grpc-io thread from the gRFC though?  If we add this as a step to follow in the README, one could easily go to the grpc-io forum and search for the proposal name to find discussion about it.

Also, I would be in favor of removing it entirely, since the system we are following today (not as documented, but what we are doing) theoretically encourages discussion in two places.  But I'm fine with leaving it as part of the process in this PR.

Reply:
I agree adding the back-reference is annoying, but I do think the need to add the backreference is what _causes_ the email to be sent to grpc-io. Without it I think it would often be forgotten. (And it won't be as obvious to the approver it is missing.)

Reply:
Re-added.